src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -37,14 +39,23 @@ public Attribute getAttributeByName(String name) {
                 .findFirst()
                 .orElse(null);
     }
+    private void findHighestAttributes() {
+        Supplier<Attribute> highestAttribute = () -> Collections.max(attributes);
+
+        for (Attribute a : attributes){
+            if (a.getValue() == highestAttribute.get().getValue()){
+                highestAttributes.add(a);
+            }
+        }
+    }
 
     public ArrayList<Attribute> getAttributes() {
         return attributes;
     }
 
     @JsonIgnore
-    public Attribute getHighestAttribute() {
-        return this.highestAttribute.get();
+    public ArrayList<Attribute> getHighestAttributes() {
```

Comment:
We don't need to know all the highest attributes. We just need to pick one.

Reply:
My thinking when I did this was we have cards like `Kaedweni_Sergeant 3 3 3 3 3` in our deck. If we only picked on attribute to make our max attribute the gameplay will be less fun, because as soon as the `Kaedweni_Sergeant` comes up, you could anticipate what attribute the AI is going to pick. The purpose of the implementation was to make the AI choices random.

Reply:
I might have misunderstood what you meant!

Reply:
You did misunderstand :)
You have defined a public Array of max attributes. We don't need it. In the end, we're just going to use ONE selected attribute. We can still make this selection random.

I can implement it, don't worry about it!